Gems/Atom/Tools/AtomToolsFramework/Code/Source/Application/AtomToolsApplication.cpp

```diff
@@ -82,12 +82,13 @@ namespace AtomToolsFramework
 
         if (auto behaviorContext = azrtti_cast<AZ::BehaviorContext*>(context))
         {
+            AZStd::string fileName = GetBuildTargetName() + ".general";
```

Comment:
This should not work because the function is virtual.  I'm surprised this would even compile.